MCMC/simulation.py

```diff
@@ -20,6 +20,7 @@ def __init__(
             r_cut=2.5,
             energy_write_freq=100,
             trajectory_write_freq=10000,
+            seed = 20
```

Comment:
We'll need a comma here after the `20`. Also, for the sake of consistent formatting, remove the spaces on each side of the `=`

Reply:
sounds good, thanks